### PR TITLE
feat: changes for new common job-manager umbrella (MAPCO-4080)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                       |
| Chore            | ✔                                                                      |

### CODE REVIEW

File: helm/templates/_resources.tpl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{/*
+Create configmap name as used by the service name label.
+*/}}
+{{- define "configmap.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "configmap" | indent 1 }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl:6]: please check that all files end ther is not EOF warning

File: helm/values.yaml [hunk 2]
```diff
@@ -49,6 +44,10 @@ cron:
   concurrencyPolicy: Forbid
   restartPolicy: Never
 
+serviceUrls:
+  jobManager: 'http://job-service-url'
```
Comment (asafMasa) [inline helm/values.yaml:48]: shouldn't it be empty so it can be overrided from shared deployments?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/values.yaml (hunk 2 above)
Comment (almog8k) [reply helm/values.yaml:48]: the shared-deployments umbrella overrides this value.

File: helm/templates/_resources.tpl (hunk 1 above)
Comment (almog8k) [reply helm/templates/_resources.tpl:6]: done

**Review by asafMasa — APPROVED** [review]

File: .github/workflows/build_and_push.yaml [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
-      - run: docker build . -t ${{ env.Docker_Repository }}
-      - run: docker push ${{ env.Docker_Repository }}
+  build_and_push_docker:
+      uses: MapColonies/shared-workflows/.github/workflows/build-and-push-docker.yaml@master
+      secrets: inherit
+  
+  build_and_push_helm:
+      uses: MapColonies/shared-workflows/.github/workflows/build-and-push-helm.yaml@master
+      secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/build_and_push.yaml:28]: Remove these whitespaces

File: .github/workflows/pull_request.yaml [hunk 4]
```diff
@@ -3,75 +3,7 @@ name: pull_request
...
-    steps:
-      - name: Check out Git repository
-        uses: actions/checkout@v2
-
-      - name: build Docker image
-        run: docker build -t test-build:latest .
+  pull_request:
+    uses: MapColonies/shared-workflows/.github/workflows/pull_request.yaml@workersTS
+    secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/pull_request.yaml:9]: Remove these whitespaces

File: .github/workflows/release-on-tag-push.yaml [hunk 5]
```diff
@@ -6,15 +6,7 @@ on:
...
-      - name: Publish Release to Github
-        uses: softprops/action-gh-release@v1
-        with:
-          body_path: CHANGELOG.md
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release_on_tag_push:
+    uses: MapColonies/shared-workflows/.github/workflows/release-on-tag-push.yaml@master
+    secrets: inherit
+
```
Comment (netanelC) [inline .github/workflows/release-on-tag-push.yaml:12]: Remove these whitespaces

File: helm/templates/configmap.yaml [hunk 6]
```diff
@@ -1,17 +1,18 @@
 {{- $chartName := include "task-liberator.name" . -}}
+{{- $configmapName := include "configmap.fullname" . }}
 {{- $tracingUrl := include "task-liberator.tracingUrl" . -}}
 {{- $metricsUrl := include "task-liberator.metricsUrl" . -}}
 {{- if .Values.enabled -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-configmap
+  name: {{ $configmapName }}
```
Comment (netanelC) [inline helm/templates/configmap.yaml:9]: Isn't it better to set the tpl to be generic instead of customize it to configmap?

**Review by netanelC — CHANGES_REQUESTED** [review]

File: helm/templates/configmap.yaml (hunk 6 above)
Comment (almog8k) [reply helm/templates/configmap.yaml:9]: After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.

File: .github/workflows/build_and_push.yaml (hunk 3 above)
Comment (almog8k) [reply .github/workflows/build_and_push.yaml:28]: done

File: .github/workflows/pull_request.yaml (hunk 4 above)
Comment (almog8k) [reply .github/workflows/pull_request.yaml:9]: done

File: .github/workflows/release-on-tag-push.yaml (hunk 5 above)
Comment (almog8k) [reply .github/workflows/release-on-tag-push.yaml:12]: done

**Review by netanelC — APPROVED** [review]

**Pull request merged**